github/activity_events.go

```diff
@@ -28,9 +28,9 @@ func (e Event) String() string {
 	return Stringify(e)
 }
 
-// Payload returns the parsed event payload. For recognized event types,
+// ParsePayload parses the event payload. For recognized event types,
 // a value of the corresponding struct type will be returned.
-func (e *Event) Payload() (payload interface{}) {
+func (e *Event) ParsePayload() (payload interface{}, err error) {
```

Comment:
So, we actually have a choice here, and I just wanna make sure we've considered both options before going through with it.

1.	We make this a breaking API change, as done above, and bump libraryVersion by one when merging.

2.	We make this a not breaking API change by adding the old method, and marking it as deprecated. It would look like this:

	```Go
	// Payload returns the parsed event payload. For recognized event types,
	// a value of the corresponding struct type will be returned.
	//
	// Deprecated: Use ParsePayload instead, which returns an error rather than panics if unmarshaling fails.
	func (e *Event) Payload() (payload interface{}) {
		var err error
		payload, err = e.ParsePayload()
		if err != nil {
			panic(err)
		}
	}
	```